recipes/msstitch/meta.yaml

```diff
@@ -18,14 +18,14 @@ build:
 
 requirements:
   host:
-    - python
-    - setuptools
+    - python >3
```

Comment:
There are no versions of python >3

Reply:
Is that not what it is supposed to look like according to: https://bioconda.github.io/linting.html#noarch-or-not-noarch ? I assumed it was to be above 3.0.

Reply:
That's a typo in the documentation, it should be `>=3`.

Reply:
No problem, thank you :)